lighthouse-core/audits/seo/canonical.js

```diff
@@ -32,7 +32,7 @@ const UIStrings = {
    * @description Explanatory message stating that there was a failure in an audit caused by a URL being relative instead of absolute.
    * @example {https://example.com/} url
    * */
-  explanationRelative: 'Relative URL ({url})',
+  explanationRelative: 'Relative URL ({url}) (not recommended)',
```

Comment:
an alternative... we communicate it's not absolute. that's what we want.

```suggestion
  explanationRelative: 'Is not an absolute URL ({url})',
```

Reply:
#12676 adopted my suggestion, so that works fine for me. :D